internal/indexmanagement/reconcile.go

```diff
@@ -140,65 +134,47 @@ func ReconcileCurationConfigmap(apiclient client.Client, cluster *apis.Elasticse
 	})
 	return errCtx.Wrap(err, "failed to update configmap")
 }
-
-func ReconcileRolloverCronjob(apiclient client.Client, cluster *apis.Elasticsearch, policy apis.IndexManagementPolicySpec, mapping apis.IndexManagementPolicyMappingSpec, primaryShards int32) error {
-	if policy.Phases.Hot == nil {
-		log.Info("Skipping rollover cronjob for policymapping; hot phase not defined", "policymapping", mapping.Name)
+func ReconcileIndexManagementCronjob(apiclient client.Client, cluster *apis.Elasticsearch, policy apis.IndexManagementPolicySpec, mapping apis.IndexManagementPolicyMappingSpec, primaryShards int32) error {
+	containers := []*corev1.Container{}
+	if policy.Phases.Delete != nil {
 		return nil
```

Comment:
accidentally left in? or is github not formatting correctly...

Reply:
no this looks to be a valid error.  will correct